templates/splash/api-en.html

```diff
@@ -18,7 +18,7 @@
 	<dt>Type</dt>
 	<dd>Canada.ca template</dd>
 	<dt>Last review</dt>
-	<dd>2022-09-28</dd>
+	<dd>2022-10-05</dd>
 
 	<dt>Specification</dt>
 	<dd>There is no specification yet</dd>
```

Comment:
Let specify the version

Like this one is a minor change because we are adding a splash page variant.

```suggestion
	<dd>There is no specification yet</dd>
	<dt>Version</dt>
	<dd>2.1</dd>
```